code/noRAM/app/build.gradle.kts

```diff
@@ -67,6 +69,16 @@ dependencies {
 
     //Photos from device
     implementation("com.github.dhaval2404:imagepicker:2.1")
+    
+    // Add the dependencies for the Firebase Cloud Messaging and Analytics libraries
+    implementation("com.google.firebase:firebase-messaging")
+    // implementation("com.google.firebase:firebase-analytics")
```

Comment:
can this commented out line be removed?

Reply:
nope I used a different method no file needed

Reply:
done